modules/installation-uninstall-aws-user-infra.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws_user_infra/uninstalling-cluster-aws-user-infra.adoc
+
+[id="installation-uninstall-aws-user-infra_{context}"]
+= Removing a cluster from AWS that uses CloudFormation templates
+
+You can remove a cluster that you installed on Amazon Web Services (AWS) by
+using CloudFormation templates.
+
+.Prerequisites
+
+* Download the AWS CLI and install it on your computer. See
+link:https://docs.aws.amazon.com/cli/latest/userguide/install-bundle.html[Install the AWS CLI Using the Bundled Installer (Linux, macOS, or Unix)]
+in the AWS documentation.
+* If you used CloudFormation templates, you have a list of the CloudFormation
+stacks that you created for your cluster.
+
+.Procedure
+
+* Delete all of the cluster resources. If you used the CloudFormation templates,
+link:https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-console-delete-stack.html[delete their stacks]:
++
+----
+$ aws cloudformation delete-stack --stack-name <name> <1>
```

Comment:
@cuppett Is there is order that needs to be followed?
looking at https://github.com/openshift/release/blob/28915b634044ebbdee1e0f7e500782c25664b202/ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml#L853-L856

Reply:
Yeah, per QE's testing, I think the order need to be followed.
1. delete stack in order: worker, master, bootstrap, security
2. Run `openshift-install destroy cluster` to release provisioned elb in the VPC to avoid the following stack deletion dependency failure.
3. delete stack in order: infra, vpc